### PR TITLE
Update ingress.md

### PR DESCRIPTION
**Pull request by kyessenov**
This fixes the stale documentation issue to fix the secure ingress task.
Confirmed manually but would help to get secondary confirmation.

### CODE REVIEW

**Comment by kyessenov** [comment]
I signed it!

**Comment by kyessenov** [comment]
fixes #552

File: _docs/tasks/traffic-management/ingress.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ to configure ingress behavior.
    
    Notice that in this example we are only exposing httpbin's `/ip` endpoint.
    
-   > Note: Envoy currently only allows a single TLS secret in the ingress since SNI is not yet supported.
+   > Note: Envoy currently only allows a single TLS secret in the ingress since SNI is not yet supported. That means that the secret name field in ingress resource is not used, and the secret must be called `istio-ingress-certs` in `istio-ingress` namespace.
```
Comment (ldemailly) [inline _docs/tasks/traffic-management/ingress.md]: you mean in istio-system namespace

File: _docs/tasks/traffic-management/ingress.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ to configure ingress behavior.
 1. Update the secret using `kubectl`
 
    ```bash
-   kubectl delete secret istio-ingress-certs; kubectl create secret tls ingress-secret --key /tmp/tls.key --cert /tmp/tls.crt
+   kubectl create secret tls istio-ingress-certs --key /tmp/tls.key --cert /tmp/tls.crt -n istio-system
```
Comment (ldemailly) [inline _docs/tasks/traffic-management/ingress.md]: as you'll rev the pr anyway, maybe move the -n closer to kubectl like we do for other examples (a bit odd to have it last arg)

File: _docs/tasks/traffic-management/ingress.md (hunk 1 above)
Comment (kyessenov) [reply _docs/tasks/traffic-management/ingress.md]: this was intentional to mean the namespace, in which istio-ingress resides, but I changed it per your suggestion.

File: _docs/tasks/traffic-management/ingress.md (hunk 2 above)
Comment (kyessenov) [reply _docs/tasks/traffic-management/ingress.md]: moved

**Comment by kyessenov** [comment]
PTAL

**Pull request merged**